src/components/Menu/MenuLinkList.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import Analytics from '../../Analytics';
+
+function MenuLinkList() {
+    const navigate = useNavigate()
+    const { t } = useTranslation(['main'])
+
+    function browseTo(path: string) {
+        Analytics.event('navigation','menu', path);
+        navigate(path)
+    }
+
+    return <ul>
+        <li onClick={()=>{browseTo("/social")}}>{t("description.perfilsocial")}</li>
```

Comment:
extract this funct into a Hoc. Move all the Menu as a module as it is now more complex